src/service.js

```diff
@@ -86,7 +86,7 @@ export function getService(win, id, opt_factory) {
  * implementation.
  * @param {!Window} win
  * @param {string} id of the service.
- * @param {function(new:T, !Window)} constructor
+ * @param {function(new:T, !Window)|function(new:T, !./service/ampdoc-impl.AmpDoc)} constructor
```

Comment:
Ditto. Not correct. Window-scoped services are always resolved against a window.